forge-gui/res/cardsfolder/upcoming/jor_kadeen_first_goldwarden.txt

```diff
@@ -0,0 +1,13 @@
+Name:Jor Kadeen, First Goldwarden
+ManaCost:R W
+Types:Legendary Creature Human Rebel
+PT:2/2
+K:Trample
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigPump | TriggerDescription$ Whenever CARDNAME attacks, it gets +X/+X until end of turn, where X is the number of equipped creatures you control. Then if CARDNAME's power is 4 or greater, draw a card.
+SVar:TrigPump:DB$ Pump | Defined$ Self | NumAtt$ X | NumDef$ X | SubAbility$ DBDraw
+SVar:DBDraw:DB$ Draw | ConditionCheckSVar$ CheckPower | ConditionSVarCompare$ GE4
```

Comment:
`ConditionDefined$ Self | ConditionPresent$ Card.powGE4`

but i think yours works too

Reply:
yeah, this would be better (`Card.powerGE4`) ☝️ because then you could also delete line 10